### PR TITLE
Updating help and associated functionality.

### PR DESCRIPTION
**Pull request by swernli**
This adds the missing help info (resolving https://github.com/Microsoft/Docker-PowerShell/issues/105) and updates existing help to use some clearer cmdlet parameters and descriptions.

@aoatkinson Could you review and make sure this looks good?  It's pretty much just what we went over together.

@jstarks FYI.

### CODE REVIEW

**Comment by jterry75** [comment]
LGTM but you need to fix the build.

**Comment by swernli** [comment]
Build fixed.  Waiting for final review by @aoatkinson.

File: src/Docker.PowerShell/Help/New-ContainerNet.md [hunk 1]
```diff
@@ -74,7 +83,7 @@ Accept wildcard characters: False
 ```
 
 ### -EnableIPv6
-{{Fill EnableIPv6 Description}}
```
Comment (aoatkinson) [inline src/Docker.PowerShell/Help/New-ContainerNet.md]: Enables IPv6 support on the network.

File: src/Docker.PowerShell/Help/New-Container.md [hunk 2]
```diff
@@ -262,6 +239,21 @@ Accept pipeline input: False
...
+### -ImageIdOrName
+The Name or Id for the image to create the new container from.
+
+```yaml
+Type: String[]
+Parameter Sets: Default
+Aliases: ImageName, ImageId
+
+Required: True
+Position: Named
```
Comment (aoatkinson) [inline src/Docker.PowerShell/Help/New-Container.md]: I think this is position 0.

**Comment by aoatkinson** [comment]
LGTM

**Comment by swernli** [comment]
@aoatkinson Thanks! I'll merge this once the latest build completes.

**Pull request merged**